modules/exploits/linux/local/pkexec.rb

```diff
@@ -66,6 +66,20 @@ def executable_path
     @executable_path
   end
 
+	def check
+		version = cmd_exec('pkexec --version')
+		if version.nil?
+			print_error "Policykit is not installed"
+			return CheckCode::Safe
+		end
+
+		if version.split('.')[1].to_f <= 0.101
```

Comment:
you can use `Gem::Version` here to make it easier.  Lots of references, check out https://github.com/rapid7/metasploit-framework/pull/7491/files#diff-715cb3460233e9e5a477394f35a977faR107